src/app/providers/login.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from '../models/User.model';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoginService {
+  loginApiUrl = `${environment.apiUrl}/${environment.loginPath}`;
+  constructor(private httpClient: HttpClient) {}
+
+  access(body: { email: string; password: string }): Promise<User> {
```

Comment:
Siamo sicuri che ritorna una Promise di User? Andrebbe creata una interfaccia nella `models folder`

tipo di dato tornato:
```
{
    token: string
    expiresIn: number,
    id: string, // non _id
    type: AuthorizationType  // creare `type AuthorizationType = 'Admin' | 'Student' | 'Teacher'` nella cartella models
}
```

Reply:
Giusto, lo faccio in vista della branch /auth/token

Reply:
Fallo qui, tanto la branch la staccherai sempre dalla feature